build.sbt

```diff
@@ -4,7 +4,7 @@ ThisBuild / startYear := Some(2002)
 ThisBuild / licenses += (("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0")))
 
 // because it doesn't declare it itself
-ThisBuild / libraryDependencySchemes += "org.scala-js" %% "scalajs-library" % "semver-spec"
+ThisBuild / libraryDependencySchemes += "org.scala-js" %% "scalajs-library" % "always"
```

Comment:
This is not the recommended way of fixing the issue.

We should remove this line now that Scala.js does declare its versioning scheme.

And we should relax the compatibility intention to `Compatibility.BinaryCompatible`.

Reply:
Ok, good to know. I didn't realize we could change that policy, and also that it would fix the dependency issue.